### ISSUE TITLE
Hosting multiplayer RCT maps prevents ride admission prices

### ISSUE THREAD

**Issue by RogoDarius** [OpenRCT2/OpenRCT2#5457]
**OS:** [e.g. Windows 10]
**Version:** [e.g. 0.0.8]
**Commit/Build:** [e.g. 426e106]

Hosting Original RCT or Added Attractions maps on multiplayer locks all ride admissions prices to "Free" (greyed out) and only allows you to change the entrance fee.
But clicking new game (singleplayer) on one of these allows you to set both (as expected).

Curiously - loading an existing singleplayer .sv4 while in multiplayer will allow rides to be changed as normal (tested with a park from original RCT1)

- [No] Reproducible in RCT2 (vanilla)?
- [Yes] Multiplayer?

**Steps to reproduce:**
1. Click "Play with other players"
2. Click "Start Server"
3. Click "New Game"
4. Choose any RCT1 or Added Attractions map
5. Observe any ride cannot have a ticket price assigned (while shops and the park entrance still work correctly)

**Screenshots / Video:**
![ivorytowers](https://cloud.githubusercontent.com/assets/12530301/25864157/4f91e896-34e6-11e7-926a-363de3caa7a0.PNG)

**Comment by Patrik356b** [comment]
Seen similar problem with Funtopia in another issue. I don't remember the issue id

**Comment by RogoDarius** [comment]
Probably: https://github.com/OpenRCT2/OpenRCT2/issues/5449
We were just trying to play RCT the way it was originally played on the old maps, but over a LAN sat next to eachother.
We got around the intermittent de-syncs easily enough by just reconnecting the client to host every once in a while. But the staff, prices, land & guest-rates all seemed to mess up on importing a .sv4 map to ORCT2.
GLaD these are being looked at though :)

**Comment by Gymnasiast** [comment]
This is inherent to SV6. It either has free rides and paid admission or the other way round.
When you import an SV4, a cheat is activated to allow paid rides and admission, but we cannot save this cheat in the game, so it will be deactivated the next time you start the game.

When we switch to our own file format we will allow RCT1-style admission and ride prices.
